packages/web/src/services/audiusQueryContext.ts

```diff
@@ -0,0 +1,21 @@
+import { AudiusQueryContextType } from '@audius/common'
+
+import { apiClient } from 'services/audius-api-client'
+import { audiusBackendInstance } from 'services/audius-backend/audius-backend-instance'
+import { audiusSdk } from 'services/audius-sdk/audiusSdk'
+import { env } from 'services/env'
+import { remoteConfigInstance } from 'services/remote-config/remote-config-instance'
+
+import { store } from '../store/configureStore'
+import { reportToSentry } from '../store/errors/reportToSentry'
+
+export const audiusQueryContext: AudiusQueryContextType = {
```

Comment:
ooh so the reason i added this was because of my handle schema - it needed to be able to pass the query context to the signup fetch apis and it isn't located within a component.  but i think i can just create a schema generator that accepts the context and the component can pass the context to the generator, ill see if that works